app/src/androidTest/java/org/simple/clinic/bp/BloodPressureRepositoryAndroidTest.kt

```diff
@@ -127,6 +128,41 @@ class BloodPressureRepositoryAndroidTest {
     assertThat(savedBloodPressure).isEqualTo(expected)
   }
 
+  @Test
+  fun observing_a_deleted_blood_pressure_as_a_stream_should_work_correctly() {
+    val now = Instant.now(clock)
+    val bloodPressure = testData.bloodPressureMeasurement(
+        createdAt = now,
+        updatedAt = now,
+        deletedAt = null,
+        syncStatus = SyncStatus.DONE)
+    appDatabase.bloodPressureDao().save(listOf(bloodPressure))
+
+    val deletedMeasurementObserver = repository.deletedMeasurementAsStream(bloodPressure.uuid)
+        .test()
```

Comment:
nitpick: can you please move `deletedMeasurementAsStream()` to the next line?

```kotlin
    val deletedMeasurementObserver = repository
        .deletedMeasurementAsStream(bloodPressure.uuid)
        .test()
```

Reply:
Fluent APIs look good when they are of >= 3 lines

Reply:
We really should get an autoformatter setup sometime.

Reply:
Fixed.